services/webdav/pkg/service/v0/service.go

```diff
@@ -435,7 +450,7 @@ func (g Webdav) sendThumbnailResponse(rsp *thumbnailssvc.GetThumbnailResponse, w
 	w.Header().Set("Content-Type", rsp.Mimetype)
 	_, err = io.Copy(w, dlRsp.Body)
 	if err != nil {
-		g.log.Error().Err(err).Msg("failed to write thumbnail to response writer")
+		logger.Error().Err(err).Msg("failed to write thumbnail to response writer")
```

Comment:
@C0rby @butonic I think this could be level warn. Could the admin do anything about it when we land in this codepath?

Reply:
could not copy stream, definitely error